relays/ethereum/src/rpc.rs

```diff
@@ -0,0 +1,338 @@
+// Copyright 2019-2020 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges Common.
+
+// Parity Bridges Common is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Bridges Common is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges Common.  If not, see <http://www.gnu.org/licenses/>.
+
+//! RPC Module
+
+#![allow(dead_code)]
+#![allow(unused_variables)]
```

Comment:
why?

Reply:
Since the no-one is using the `SubstrateRpcClient` and `EthereumRpcClient` structs I get a ton of warnings about unused/dead code. This would be removed once integration happens as part of #72.